adafruit_featherwing/joy_featherwing.py

```diff
@@ -0,0 +1,284 @@
+              :alt: Joy FeatherWing Button B
+
+            This example prints when button B is pressed.
+
+            .. code-block:: python
+
+                from adafruit_featherwing import joy_featherwing
+                import time
+
+                WING = joy_featherwing.JoyFeatherWing()
+
+                while True:
+                    if WING.button_b:
+                    print("Button B pressed!")
+
+        """
+        return self._check_button(BUTTON_B)
+
+    @property
+    def button_x(self):
+        """Joy featherwing button X.
+
+            .. image :: /_static/joy_featherwing/joy_featherwing_x.jpg
+              :alt: Joy FeatherWing Button X
+
+            This example prints when button X is pressed.
+
+            .. code-block:: python
+
+                from adafruit_featherwing import joy_featherwing
+                import time
+
+                WING = joy_featherwing.JoyFeatherWing()
+
+                while True:
+                    if WING.button_x:
+                    print("Button X pressed!")
+
+        """
+        return self._check_button(BUTTON_X)
+
+    @property
+    def button_y(self):
+        """Joy featherwing button Y.
+
+            .. image :: /_static/joy_featherwing/joy_featherwing_y.jpg
+              :alt: Joy FeatherWing Button Y
+
+            This example prints when button Y is pressed.
+
+            .. code-block:: python
+
+                from adafruit_featherwing import joy_featherwing
+                import time
+
+                WING = joy_featherwing.JoyFeatherWing()
+
+                while True:
+                    if WING.button_y:
+                    print("Button Y pressed!")
+
+        """
+        return self._check_button(BUTTON_Y)
+
+    @property
+    def button_select(self):
+        """Joy featherwing button SELECT.
+
+            .. image :: /_static/joy_featherwing/joy_featherwing_select.jpg
+              :alt: Joy FeatherWing Button SELECT
+
+            This example prints when button SELECT is pressed.
+
+            .. code-block:: python
+
+                from adafruit_featherwing import joy_featherwing
+                import time
+
+                WING = joy_featherwing.JoyFeatherWing()
+
+                while True:
+                    if WING.button_select:
+                    print("Button SELECT pressed!")
+
+        """
+        return self._check_button(BUTTON_SELECT)
+
+    def _check_button(self, button):
+        """Utilises the seesaw to determine which button is being pressed."""
+        buttons = self._seesaw.digital_read_bulk(button)
+        return not buttons != 0
+
+    @property
+    def joystick_offset(self):
+        """Returns the joystick offset, set by the `joystick_offset.setter`."""
+        return self._joystick_offset
+
+    @joystick_offset.setter
+    def joystick_offset(self, offset):
+        """Allows for setting explicit numbers to effectively zero the joystick.
```

Comment:
Move this comment up to the getter. The setter won't have one.

I'd make the first sentence:
`Offset used to correctly report (0, 0) when the joystick is centered.`